### ISSUE TITLE
Tooltips misplaced when using hvPlot/ HoloViews/ Bokeh plot in Fast template.

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#5778]
panel=1.3.0, hvplot=0.9.0, holoviews=1.18.0, bokeh=3.3.0

![image](https://github.com/holoviz/holoviews/assets/42288570/53c45898-0841-431d-8866-28ebc9f88d1d)

```python
import pandas as pd
import panel as pn
import hvplot.pandas

pn.extension()

data = pd.DataFrame(
    dict(x=list(range(10)), y=list(range(10)))
)

component = data.hvplot(x="x", y="y")

pn.template.FastListTemplate(main=[component]).servable()
```

If I change to Material template or Bokeh 3.2.2 there is no issue.

## Additional Context

Most of our Panel apps use the combination of Fast and hvplot/ HoloViews/ Bokeh. Thus we cannot upgrade to the current versions of Bokeh.

## Workarounds

- Change to for example MaterialTemplate or
- Downgrade from Bokeh 3.3.0 to Bokeh 3.2.2. FYI. @mattpap .

**Comment by mattpap** [comment]
Does fast template contain any `contain: strict` or similar CSS? If so, this is fixed by https://github.com/bokeh/bokeh/pull/13431.

**Comment by MarcSkovMadsen** [comment]
Not that I am aware of or can find by inspection. I suspect somehow the `<fast-card>` element is the cause.

![image](https://github.com/holoviz/panel/assets/42288570/9658cc43-9dba-400c-8f34-363aa55586cc)

**Comment by MarcSkovMadsen** [comment]
Its the [`<fast-card>`](https://explore.fast.design/components/fast-card) that is the cause. If I remove it via by setting `main_layout=None`, then there is no problem.

```python
import pandas as pd
import panel as pn
import hvplot.pandas

pn.extension()

data = pd.DataFrame(
    dict(x=list(range(10)), y=list(range(10)))
)

component = data.hvplot(x="x", y="y")

pn.template.FastListTemplate(main=[component], main_layout=None).servable()
```

**Comment by philippjfr** [comment]
Indeed fixed by Bokeh 3.3.1, so I'll close.
